src/utils/jwt_config.py

```diff
@@ -0,0 +1,15 @@
+from flask_jwt_extended import JWTManager
+
+def init_jwt(app):
+    """Inicializa la configuración de JWT"""
+    jwt = JWTManager(app)
+    
+    @jwt.additional_claims_loader
+    def add_role_to_token(identity):
+        """Agrega el rol del usuario al token JWT"""
+        return {
+            "role": identity.get("role", "student"),
+            "user_id": identity.get("id")
+        }
```

Comment:
Creo que no podrías usar `get` directamente. Si `identity` es el token de acceso en la cabecera de la petición entonces es una cadena `str`. Deberías hacer una conversión primero.

```suggestion
        import json
        json_identity = json.loads(identity)
        
        return {
            "role": json_identity.get("role", "student"),
            "user_id": json_identity.get("id")
        }
```